### PR TITLE
Two Columns Layout prettify

### PR DESCRIPTION
**Pull request by harkiratsm**
Signed-off-by: harkiratsm <multaniharry714@gmail.com>

**Description**

This PR fixes #

**Notes for Reviewers**

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [ ] Yes, I signed my commits.

### CODE REVIEW

File: ui/themes/rjsf.js [hunk 1]
```diff
@@ -30,15 +30,15 @@ export const rjsfTheme = createTheme({
...
       root : {
-        width : '70%',
+        marginRight : "0.5rem"
       }
     },
     MuiInputLabel : {
       root : {
         whiteSpace : "nowrap",
-        overflow : "hidden",
+        // overflow : "hidden",
```
Comment (harkiratsm) [inline ui/themes/rjsf.js:41]: This Comment is on purpose would fix it.

File: ui/components/MesheryMeshInterface/PatternService/RJSFCustomComponents/CustomUpDownWidget.js [hunk 2]
```diff
@@ -8,8 +8,8 @@ const style={
 const CustomHelperText=(props) => {
   return (
     <div style={style} id={props.id}>
-      <ErrorIcon style={{ color : "red",marginRight : "2px",height : "18px" }}/>
-      <FormHelperText error>{props.errormsg}</FormHelperText>
+      <ErrorIcon style={{ color : "#B32700",marginRight : "0.2rem",height : "1rem" }}/>
```
Comment (leecalcote) [inline ui/components/MesheryMeshInterface/PatternService/RJSFCustomComponents/CustomUpDownWidget.js:11]: Thank you, @harkiratsm :100

File: ui/components/MesheryMeshInterface/PatternService/RJSFCustomComponents/CustomUpDownWidget.js [hunk 3]
```diff
@@ -20,16 +20,19 @@ const CustomUpDownField = (props) => {
...
     <>
       <TextField
-        id="standard-number"
+        id={props.id}
+        size="small"
         label={name}
         key={props.id}
         value={props?.value}
-        variant="standard"
+        variant="outlined"
```
Comment (leecalcote) [inline ui/components/MesheryMeshInterface/PatternService/RJSFCustomComponents/CustomUpDownWidget.js:28]: 👍

**Comment by leecalcote** [comment]
@harkiratsm did you review how this looks in Meshery UI Designer, too?

**Review by leecalcote — APPROVED** [review]

**Comment by harkiratsm** [comment]
@leecalcote donot merge it until I give flag, as i need to do more changes.

**Comment by harkiratsm** [comment]
@leecalcote :+1:

**Comment by leecalcote** [comment]
Good to go…

File: ui/themes/rjsf.js (hunk 1 above)
Comment (leecalcote) [reply ui/themes/rjsf.js:41]: Does this need undone? Does the textOverflow : "ellipsis" below not take effect appropriately? It didn't seem to on today's call.

Comment (leecalcote) [reply ui/themes/rjsf.js:41]: Needs fixed, I think.

**Pull request merged**